osu.Game/Beatmaps/Beatmap.cs

```diff
@@ -47,6 +47,21 @@ public class Beatmap<T>
         public Storyboard Storyboard = new Storyboard();
 
         /// <summary>
+        /// Whether this beatmap's background should be hidden while its storyboard is being displayed.
+        /// </summary>
+        public bool StoryboardReplacesBackground

```

Comment:
Going forward we cannot be adding the `Storyboard` to the `Beatmap`, thus I would like this to be moved into `Storyboard`.